docs/src/gui/gmoccapy.adoc

```diff
@@ -14,34 +14,34 @@
 
 == Introduction
 
-'GMOCCAPY' is a GUI for LinuxCNC, designed to be used with a touch screen,
+GMOCCAPY is a GUI for LinuxCNC, designed to be used with a touch screen,
 but can also be used on normal screens with a mouse or hardware buttons and MPG
 wheels, as it presents HAL Pins for the most common needs. Please find more
 information in the following.
 
 It offers the possibility to display up to 9 axis, support a lathe mode for
 normal and back tool lathe and can be adapted to nearly every need, because
-gmoccapy supports embedded tabs and side panels.
+GMOCCAPY supports embedded tabs and side panels.
```

Comment:
Why uppercase letters here?

Not sure which is @gmoccapy's preferred spelling...